tests/test_models.py

```diff
@@ -460,3 +483,62 @@ class TestModel(models.Model):
     __test__ = False  # this is not a test
     value = models.IntegerField(null=True)
     other = models.IntegerField(null=True)
+
+
+class TestAuditEventBootstrapping(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.aerodrome_details = {
+            "KIAD": 313,
+            "VIDP": 777,
+            "FACT": 151,
+        }
+        for icao, amsl in cls.aerodrome_details.items():
+            Aerodrome.objects.create(
+                icao=icao,
+                elevation_amsl=amsl,
+                amsl_unit="ft",
+            )
+
+    def test_bootstrap_existing_model_records(self):
+        with patch.object(AuditEvent.objects, "bulk_create",
+                          side_effect=AuditEvent.objects.bulk_create) as mock:
+            self._verify_bootstrap_existing_model_records(None)
+            mock.assert_called_once_with(ANY)
```

Comment:
I have mixed feelings about helper methods that wrap assertions because I think it makes tests harder to reason with, but that's just a personal preference. I do however feel strongly that the actual method under test (`bootstrap_existing_model_records` in this case) should be called within the test itself. This also would make the verify method easier to reason with.

Reply:
I refactored this a bit. I still have a helper method that wraps assertions to avoid lots of duplication, but I've pulled the method under test (and a few other related things) out to hopefully make it easier to follow.